src/Sulu/Component/Util/XmlUtil.php

```diff
@@ -21,10 +21,8 @@ class XmlUtil
      *
      * @param string $path
      * @param \DOMNode $context
-     *
-     * @return bool|null|string|mixed
      */
-    public static function getValueFromXPath($path, \DOMXPath $xpath, \DOMNode $context = null, $default = null)
+    public static function getValueFromXPath($path, \DOMXPath $xpath, \DOMNode $context = null, $default = null): mixed
     {
         $result = $xpath->query($path, $context);
```

Comment:
What if result is null and the node couldn't be found? I think currently this should throw an error because calling `->length` on false is not allowed. There should probably be an assert here if we want to have a better error message or just return a the default?

Reply:
I have solved it by just returning the default value if the XML node was not found.